pkg/virt-launcher/virtwrap/network/common.go

```diff
@@ -76,10 +76,11 @@ func (e *CriticalNetworkError) Error() string { return e.Msg }
 
 func (vif VIF) String() string {
 	return fmt.Sprintf(
-		"VIF: { Name: %s, IP: %s, Mask: %s, MAC: %s, Gateway: %s, MTU: %d, IPAMDisabled: %t, TapDevice: %s}",
+		"VIF: { Name: %s, IP: %s, Mask: %s, IPv6: %s, MAC: %s, Gateway: %s, MTU: %d, IPAMDisabled: %t, TapDevice: %s}",
 		vif.Name,
 		vif.IP.IP,
 		vif.IP.Mask,
+		vif.IPv6,
```

Comment:
Why IPv4 is using `IP.IP` and IPv6 is not using `IPv6.IP` (+ Mask)?

Reply:
Since unlike ipv4, ipv6 may be `nil`. To avoid special code checking if it is `nil`vif.IPv6 is used.